docs/pelion_ready/device-management-config.md

```diff
@@ -2,47 +2,47 @@
 
 Device Management configuration has five distinct areas:
 
-- Connectivity - the transport type for the device to connect to the device management service.
+- Connectivity - the transport type for the device to connect to the Device Management service.
 - Storage - the storage type and writing used for both the credentials and the firmware storage.
-- Flash geometry - the device flash "sandbox" for bootloader, application header, application image and two SOTP regions.
+- Flash geometry - the device flash "sandbox" for bootloader, application header, application image and two Software One Time Programming (SOTP) regions.
```

Comment:
One-Time (capitalization?)

Reply:
We have this without in our glossary. However, the glossary may be wrong.

Reply:
@AnotherButler, this one's for you.